sdk/tables/azure-data-tables/samples/tables_entity_operations.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <azure/data/tables.hpp>
+
+#include <cstdio>
+#include <iostream>
+#include <stdexcept>
+#include <thread>
+
+std::string GetConnectionString()
+{
+  const static std::string ConnectionString = "";
+
+  if (!ConnectionString.empty())
+  {
+    return ConnectionString;
+  }
+  const static std::string envConnectionString = std::getenv("STANDARD_STORAGE_CONNECTION_STRING");
```

Comment:
Same comment - unless you could use the very same azure connection string to run Storage samples and Tables samples, rename it to `DATA_TABLES_CONNECTION_STRING`.

Reply:
Yes i connect to the storage accoutn, it's the same conn string that could be used by blob service for example